src/img/icons/icon-profile.svg

```diff
@@ -1,3 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 35 35">
-  <path fill="#535353" fill-rule="evenodd" d="M34.657 17.5c0 9.608-7.755 17.363-17.363 17.363S0 27.04 0 17.5 7.755.137 17.363.137 34.657 7.892 34.657 17.5zm-17.294-.137c2.47 0 4.53-4.05 4.53-6.588 0-2.47-2.06-4.53-4.53-4.53s-4.53 2.06-4.53 4.53 2.06 6.588 4.53 6.588zm9.127 6.588c-.068-.066.206-5.693-2.333-6.38-1.304-.343-2.402-.55-2.95-.686-.893 1.167-2.334 2.265-3.913 2.265-1.647 0-3.02-1.03-3.912-2.266-.617.138-1.715.412-2.95.687-2.47.617-2.197 6.725-2.197 6.725C10.5 26.627 13.657 28 17.157 28c3.706 0 7-1.578 9.333-4.05z"/>
-</svg>
+<svg width="20" height="22" viewBox="0 0 12 13" xmlns="http://www.w3.org/2000/svg"><path d="M2.67 11.33V9a.33.33 0 0 1 .66 0v2.67c0 .18-.15.33-.33.33H1.67a1 1 0 0 1-1-1V9.67c0-2.6 1.8-4.34 5.33-4.34 3.53 0 5.33 1.74 5.33 4.34V11a1 1 0 0 1-1 1H9a.33.33 0 0 1-.33-.33V9a.33.33 0 1 1 .66 0v2.33h1c.19 0 .34-.15.34-.33V9.67C10.67 7.44 9.17 6 6 6S1.33 7.44 1.33 9.67V11c0 .18.15.33.34.33h1zm3.33-6c1.09 0 2-1.02 2-2.33C8 1.7 7.09.67 6 .67 4.91.67 4 1.69 4 3c0 1.3.91 2.33 2 2.33zM6 6C4.53 6 3.33 4.66 3.33 3S4.53 0 6 0s2.67 1.34 2.67 3S7.47 6 6 6zm2.67 3.67a.33.33 0 1 1 .66 0v2c0 .09-.03.17-.1.23-.12.12-.37.28-.8.42-.65.22-1.56.35-2.76.35-.92 0-1.65-.14-2.21-.36-.36-.14-.58-.3-.7-.4a.33.33 0 0 1-.1-.24v-2a.33.33 0 1 1 .67 0v1.84a5.33 5.33 0 0 0 2.33.5c1.14-.01 1.98-.13 2.57-.33.2-.07.35-.13.44-.18V9.67z" fill="#266ABD" fill-rule="nonzero"/></svg>
```

Comment:
Does this icon get overwritten colour wise anywhere? Currently blue in your screenshot.

Reply:
fixed

Reply:
added colour with css